client/src/components/Form/Form.jsx

```diff
@@ -95,8 +127,152 @@ const Form = () => {
           onChange={handleDateChange}
         />
       </MuiPickersUtilsProvider>
+
+      <TextField
+        fullWidth
```

Comment:
Should each of the form fields be able to read from the search_form config to add the * in the UI next to the fields that are actually indicated as required?

Currently, if you make additional fields required besides first name, last name and DOB, there isn't really a way for the user to be able to tell which fields are necessarily required. 

Maybe we can add a `required` attribute to each of the field elements, add some logic to set it to true whenever that field is actually required.